.dryrunsecurity.yaml

```
@@ -62,6 +63,10 @@ allowedAuthors:
     - cneill
     - Maffooch
     - blakeowens
```

To make the fix easy 😄 

```suggestion
    - blakeaowens
```